tsconfig.json

```diff
@@ -15,7 +15,8 @@
     "noUnusedParameters": true,
     "strictNullChecks": true,
     "strictFunctionTypes": true,
-    "noFallthroughCasesInSwitch": true
+    "noFallthroughCasesInSwitch": true,
+    "strictPropertyInitialization": false,
```

Comment:
I prefer using `?` at class member fields if really needed.

Reply:
Yeah! I was hesitant on this one, typescript can be a real pain sometimes.

Reply:
So would you mind reverting tsconfig and using `!`?